### PR TITLE
feat: manage the secrets annotated to be managed by the controller

### PR DESCRIPTION
**Pull request by malmuzaini**

### CODE REVIEW

File: cmd/controller/controller.go [hunk 1]
```diff
@@ -61,6 +61,10 @@ type Controller struct {
 	oldGCBehavior bool // feature flag to revert to old behavior where we delete the secrets instead of relying on owners reference.
 }
 
+// Annotation flag key that the controller will check on the found Secret
+// objects to decide if it's will take over and manage it or not.
+const ManagedAnnotation = "sealedsecrets.bitnami.com/managed"
```
Comment (mkmik) [inline cmd/controller/controller.go]: Let's avoid hard-coding the annotation prefix. Instead, let's export the annotation prefix from https://github.com/bitnami-labs/sealed-secrets/blob/42b5c465bf75edc4f9003b23b07b66ceda5e2db0/pkg/apis/sealed-secrets/v1alpha1/types.go#L15 and use it here

File: README.md [hunk 2]
```diff
@@ -290,6 +290,9 @@ that only the intended `SealedSecret` is uploaded to the cluster.  The
 only change from existing Kubernetes is that the *contents* of the
 `Secret` are now hidden while outside the cluster.
 
+### Managing existing secrets
+If you want `SealedSecret` controller to take management of an existing `Secret`, then you have to annotate that `Secret` with the annotation `sealedsecrets.bitnami.com/managed: true` ahead applying the [Usage](#USage) steps.
```
Comment (mkmik) [inline README.md]:
```suggestion
If you want `SealedSecret` controller to take management of an existing `Secret` (i.e. overwrite it when unsealing a `SealedSecret` with the same name and namespace), then you have to annotate that `Secret` with the annotation `sealedsecrets.bitnami.com/managed: true` ahead applying the [Usage](#usage) steps.
```

**Comment by mkmik** [comment]
Integration tests are failing.
You can run them locally with:

```sh
make integrationtest
```

File: integration/controller_test.go [hunk 3]
```diff
@@ -230,6 +235,29 @@ var _ = Describe("create", func() {
 		})
 	})
 
+	Describe("Secret already exists", func() {
+		Context("With managed annoation", func() {
```
Comment (mkmik) [inline integration/controller_test.go]: minor typo

```suggestion
		Context("With managed annotation", func() {
```

**Comment by malmuzaini** [comment]
> ```shell
> make integrationtest
> ```

**Comment by malmuzaini** [comment]
> Integration tests are failing.
> You can run them locally with:
> 
> ```shell
> make integrationtest
> ```

Yes I did I told it's failing I don't what's the issue with my assertion. But let me check try second time. And I will get back to you. if I needed a help

**Pull request merged**